stacktrace-gps.js

```diff
@@ -243,14 +251,24 @@
                 this._get(fileName).then(function (source) {
                     var sourceMappingURL = _findSourceMappingURL(source);
                     var isDataUrl = sourceMappingURL.substr(0, 5) === 'data:';
+                    var base = fileName.substring(0, fileName.lastIndexOf('/') + 1);
 
                     if (sourceMappingURL[0] !== '/' && !isDataUrl && !(/^https?:\/\/|^\/\//i).test(sourceMappingURL)) {
-                        sourceMappingURL = fileName.substring(0, fileName.lastIndexOf('/') + 1) + sourceMappingURL;
+                        sourceMappingURL = base + sourceMappingURL;
                     }
 
                     this._get(sourceMappingURL).then(function (map) {
                         var lineNumber = stackframe.lineNumber;
                         var columnNumber = stackframe.columnNumber;
+
+                        if (typeof map === 'string') {
+                            map = _parseJson(map.replace(/^\)\]\}'/, ''));
```

Comment:
What's the need for running `map.replace()` here? When would one encounter such a beginning of a line (literally `)]}'`)?